### ISSUE TITLE
`getAllMessagesInChat` is returning a message with `sender` as String, while it should be an object (per documentation)

### ISSUE THREAD

**Issue by feliperaul** [open-wa/wa-automate-nodejs#610]
**Describe the bug**
After #590 was fixed, `getAllMessagesInChat` now returns an array of objects again.

However, those objects were supposed to be Message objects documented [here](https://open-wa.github.io/wa-automate-nodejs/interfaces/message.html), and [per the documentation](https://open-wa.github.io/wa-automate-nodejs/interfaces/message.html#sender), the `sender` key should be an Object containing, amongst other fields, the `isMe` boolean; **however**, `getAllMessagesInChat` is returning an array of objects that `sender` key is only a string, like `sender: '554412345@c.us'` 

**Expected behavior**
`getAllMessagesInChat` should return an array of Message objects, that the `sender` attribute is an object that contains the `isMe` boolean. 

**DEBUG INFO**

```
┌─────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│       (index)       │                                                                    Values                                                                    │
├─────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│     WA_VERSION      │                                                                  '2.2027.9'                                                                  │
│       PAGE_UA       │ 'WhatsApp/0.4.315 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36' │
│ WA_AUTOMATE_VERSION │                                                                   '1.9.91'                                                                   │
│   BROWSER_VERSION   │                                                            'Chrome/83.0.4103.116'                                                            │
└─────────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

**Host (please complete the following information):**
 - OS: iOS

**Comment by smashah** [comment]
Docs are out of date in that case. I'm working towards a unified approach to IDs. Going forward all will be returning serialised instead of the WID object

**Comment by dantenol** [comment]
getAllMessagesInChat also throws an error when trying to read messages on a non-contact chat.

 'Uncaught SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at Object.window.WAPI.quickClean (/home/dante/dev/zoppy/server/node_modules/@open-wa/wa-automate/dist/lib/wapi.js:2226)
    at Object.window.WAPI.getAllMessagesInChat (/home/dante/dev/zoppy/server/node_modules/@open-wa/wa-automate/dist/lib/wapi.js:717)
    at <anonymous>:1:6`

**Comment by feliperaul** [comment]
@danielcardeenas I had that error as well, but later couldn't reproduce it.

@smashah You mean there's no plan to revert the old behavior, that `message` was an object?

**Comment by smashah** [comment]
Sending the WID object to the front end is pretty redundant and inefficient so no won't be reverting back to sending the whole ID object, just the ID string

**Comment by feliperaul** [comment]
@smashah Ok, could you point what would be the easiest way to get that same WID object having the ID string?

**Comment by smashah** [comment]
@feliperaul you actually use the WID object?

**Comment by smashah** [comment]
@dantenol that issue happens because there is a notification in the chat log. fix on the way

**Comment by smashah** [comment]
The error is fixed in 1.9.92 (https://github.com/open-wa/wa-automate-nodejs/issues/612#issuecomment-652999467) sender WID will not be reimplemented

**Comment by feliperaul** [comment]
@smashah Thanks, as usual I will try the fix soon and report back if find anything else. Cheers.

**Comment by dantenol** [comment]
@smashah getAllMessagesinChat works but it skips the very last message. Should I create a new issue?

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.94](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.93...1.9.94) (2020-07-02)

- Release 1.9.94 [`a46cfe5`](https://github.com/open-wa/wa-automate-nodejs/commit/a46cfe5d013bf0d6379cf943b17d5ce37e176ad4)
- Improvement to `getAllMessagesInChat` #610 [`399a329`](https://github.com/open-wa/wa-automate-nodejs/commit/399a3296c4562b9d56f354aa1544172eaf8b7024)

**Comment by smashah** [comment]
@dantenol try that
